src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -104,6 +104,7 @@ def __init__(self):
         super().__init__()
 
         self.db_conn = Namespace.init_namespace_dbs()
+        Namespace.connect_all_dbs(self.db_conn, mibs.APPL_DB)
```

Comment:
>Namespace.connect_all_dbs(self.db_conn, mibs.APPL_DB) [](start = 8, length = 53)

Original code could tolerate temporary or initial Redis not offline and recover whenever online. But the new code could not. Can you improve and test?

Reply:
I do see that if redis-server process is stopped and started, the MIB walk does not get affected, as the connect is persistent_connect. But, I have reverted this change, I will raise a new PR to provide a clean fix to reduce runtime.